specs/electra/beacon-chain.md

```diff
@@ -695,9 +704,14 @@ def queue_excess_active_balance(state: BeaconState, index: ValidatorIndex) -> No
     if balance > MIN_ACTIVATION_BALANCE:
         excess_balance = balance - MIN_ACTIVATION_BALANCE
         state.balances[index] = MIN_ACTIVATION_BALANCE
-        state.pending_balance_deposits.append(
-            PendingBalanceDeposit(index=index, amount=excess_balance)
-        )
+        validator = state.validators[index]
+        state.pending_deposits.append(PendingDeposit(
+            pubkey=validator.pubkey,
+            withdrawal_credentials=validator.withdrawal_credentials,
+            amount=excess_balance,
+            signature=bls.G2_POINT_AT_INFINITY,
+            slot=GENESIS_SLOT,
```

Comment:
The protocol does not accept a block proposed in the `GENESIS_SLOT` because of the following check in the `process_block_header`:
```python
assert block.slot > state.latest_block_header.slot
```